docs/administration-guide/databases/oracle.md

```diff
@@ -12,25 +12,24 @@ Head to this page, accept the license agreement, and download `ojdbc8.jar`:
 
 Before downloading this JAR you may need to sign up for a free account with Oracle. We have had success with the latest version at the time of this writing, 19.3 (even with older Oracle 12c databases), but any version _should_ work.
 
-### Adding the Oracle JDBC Driver JAR to the Metabase Plugins Directory
+## Adding the Oracle JDBC Driver JAR to the Metabase Plugins Directory
 
 Metabase will automatically make the Oracle driver available if it finds the Oracle JDBC driver JAR in the Metabase plugins directory when it starts up.
 All you need to do is create the directory, move the JAR you just downloaded into it, and restart Metabase.
 
-### Connecting with SSL
+## Connecting with SSL
 
-To connect to Oracle via SSL and enable encryption, check the `Use a secure connection (SSL)` option on the connection
-setup page.  You can add other SSL features (including client and/or server authentication) as explained below. You can
+To connect to Oracle via SSL and enable encryption, check the `Use a secure connection (SSL)` option on the connection setup page. You can add other SSL features (including client and/or server authentication) as explained below. You can
 use both client and server authentication (known as mutual authentication).
-
-#### Server authentication
+ 
+### Server authentication with a truststore
 
 To configure the client (Metabase) to authenticate the identity of the server (the Oracle server), you may need to
 configure a truststore file that includes the server's root CA, so that the JVM running Metabase trusts its
 certificate chain. Refer to the
-[Oracle documentation](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html) on using `keytool` to
-manage key and truststore files, importing certificates, etc. Once you have a truststore file ready, add the
-following JVM options for Metabase:
+[Oracle documentation](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html) on using `keytool` to manage key and truststore files, importing certificates, etc.
+
+Once you have a truststore file ready, add the following JVM options for Metabase:
```

Comment:
We may want to ditch this part, since now the trust store can be configured on a per-database basis (via the connection option) rather than at the JVM level.